dotCMS/src/integration-test/java/com/dotcms/experiments/business/ExperimentAPIImpIT.java

```diff
@@ -247,9 +251,32 @@ public void testADeleteOriginalVariant_shouldFail() throws DotDataException, Dot
 
     /**
      * Method to test: {@link ExperimentsAPIImpl#getEvents(Experiment)}
+<<<<<<< HEAD
+     * When: You have 4 pages let call them: A, B, C and D and:
+     * - We create a Experiment with the B page.
+     * - We mock in the test 4 Browser Session with different lookBackWindows, each of this session
+     * navigate by the follow pages.
+     * Session 1: A, B, D and C
+     * Session 2: D and C
+     * Session 3: A and B
+     * Session 4: A, B D and C
+     *
+     * All these navigation cases trigger a pageview Event for each page, so we should got these number
+     * of events for each session:
+     *
+     * Session 1: 4
+     * Session 2: 2
+     * Session 3: 2
+     * Session 4: 4
+     *
+     * If we call the {@link ExperimentsAPIImpl#getEvents(Experiment)} now
+     *
+     * Should: Return 4 {@link BrowserSession} each one with the right numbers of {@link com.dotcms.experiments.business.result.Event}
+=======
```

Comment:
fix this conflict

Reply:
done https://github.com/dotCMS/core/pull/24034/commits/b52ca2d44e2fb8ecb90366b53fbfd1db1d1ef832